tests/suites/test_suite_aes.function

```diff
@@ -289,6 +289,58 @@ exit:
 }
 /* END_CASE */
 
+/* BEGIN_CASE depends_on:MBEDTLS_CIPHER_MODE_OFB */
+void aes_encrypt_ofb( int fragment_size, char *hex_key_string,
+                      char *hex_iv_string, char *hex_src_string,
+                      char *hex_dst_string )
+{
+    unsigned char key_str[100];
```

Comment:
I know this is the same as existing tests, but I think for new tests we should pick buffer sizes more carefully and avoid using more stack than we need.

The most obvious offenders are `key_str` and `iv_str` which clearly will never require 100 bytes, but since in practice we have a limited set of test vectors, we could adjust the size of the other buffers to that of the largest test vector.

Reply:
Done.